internal/status/status.go

```diff
@@ -91,11 +96,79 @@ func GetStatus(w http.ResponseWriter, req *http.Request) {
 		lastUpdate = time.Now()
 	}
 
-	user, _ := auth.FromRequest(req)
-	if user != nil {
-		response["user"] = map[string]interface{}{
-			"user_id":      user.ID,
-			"assigned_sdk": auth.SDKAddress(user),
+	responses.AddJSONContentType(w)
+	w.WriteHeader(respStatus)
+	respByte, err := json.MarshalIndent(response, "", "  ")
+	if err != nil {
+		logger.Log().Error(err)
+	}
+	w.Write(respByte)
+}
+
+func GetStatusV2(w http.ResponseWriter, r *http.Request) {
+	respStatus := http.StatusOK
+	var response statusResponse
+
+	if cachedResponse != nil && lastUpdate.After(time.Now().Add(statusCacheValidity)) {
+		copier.Copy(&response, cachedResponse)
+	} else {
```

Comment:
instead of this giant else, i would put a `return` after `copier.Copy(&response, cachedResponse)`. this way you avoid an extra level of nested braces. its the recommended style in go: https://golang.org/doc/effective_go.html#if